services/ui-src/src/components/pages/GetStarted/McparGetStartedPage.tsx

```diff
@@ -50,6 +64,47 @@ export const McparGetStartedPage = () => {
               <Text sx={sx.additionalInfo}>{section2.additionalInfo}</Text>
             </Box>
           </Flex>
+          {section2.table && (
+            <Flex sx={sx.sectionContent}>
+              <Box>
+                <Heading sx={sx.smallSectionHeading} size={"sm"}>
+                  {section2.tableHeading}
+                </Heading>
+                <TableContainer>
+                  <Table>
+                    <Thead sx={sx.tableHeading}>
+                      <Tr>
+                        <Th>Excel Workbook Tab</Th>
+                        <Th>Location in MCPAR online form</Th>
+                      </Tr>
+                    </Thead>
+                    <Tbody sx={sx.tableBody}>
+                      {section2.table.map(
+                        (row: Record<"tab" | "location", string>) => {
+                          return (
+                            <Tr>
+                              <Td>{row.tab}</Td>
+                              <Td>{row.location}</Td>
+                            </Tr>
+                          );
+                        }
+                      )}
+                    </Tbody>
+                  </Table>
```

Comment:
Question: Does our internal `Table.tsx` component not work for this use case?

Reply:
Didn't know that existed 😎